### DISCUSSION TITLE
How to toggle trouble window with current mode in v3?

### DISCUSSION THREAD

**Discussion by eklmv** [folke/trouble.nvim#535] [Q&A]
If i understand correctly, v3 trouble requires you to specify which mode you want to toggle and there no builtin way to create a key bind which will just toggle whatever mode you currently (last one) have open? Same if for example i will change severity with `s`, i can't just toggle that trouble window back and forth maintaining chosen severity?

**Comment by agaro1121** [comment]
I used to use this feature a lot as well. Hoping there's a way to bring it back.
